src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs

```diff
@@ -236,16 +252,12 @@ internal async Task InvokeJobFunction(DateTime invocationTime, bool isPastDue =
 
             try
             {
-                FunctionResult result = await _executor.TryExecuteAsync(input, token);
-                if (!result.Succeeded)
```

Comment:
Cleanup - no need to inspect then throw, if we're just swallowing the exception anyways